package.json

```diff
@@ -126,4 +126,4 @@
       "pre-push": "npm run pre-push-message && npm run test"
     }
   }
-}
+}
```

Comment:
![image](https://user-images.githubusercontent.com/6245927/84559211-6d412500-ad56-11ea-9292-cf744ff02c1d.png)
Please fix this

Reply:
what does this mean?

Reply:
@AkalUstat at the end of the file, add a new empty line.